myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/MyriadDriver.java

```diff
@@ -15,83 +15,25 @@
  */
 package com.ebay.myriad.scheduler;
 
-import com.ebay.myriad.configuration.MyriadConfiguration;
-import com.ebay.myriad.state.SchedulerState;
-import com.google.protobuf.ByteString;
-import com.google.protobuf.InvalidProtocolBufferException;
-import org.apache.commons.lang.StringUtils;
-import org.apache.mesos.MesosSchedulerDriver;
-import org.apache.mesos.Protos.Credential;
-import org.apache.mesos.Protos.FrameworkID;
-import org.apache.mesos.Protos.FrameworkInfo;
-import org.apache.mesos.Protos.FrameworkInfo.Builder;
+import javax.inject.Inject;
+
 import org.apache.mesos.Protos.Status;
 import org.apache.mesos.Protos.TaskID;
+import org.apache.mesos.SchedulerDriver;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.inject.Inject;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.util.concurrent.ExecutionException;
-
 /**
- * The driver for the Myriad scheduler
+ * Driver for Myriad scheduler.
  */
 public class MyriadDriver {
     private static final Logger LOGGER = LoggerFactory.getLogger(MyriadDriver.class);
 
-    private final MesosSchedulerDriver driver;
+    private final SchedulerDriver driver;
```

Comment:
What is the point of switching from MesosSchedulerDriver to the interface: SchedulerDriver? You can Mock MesosSchedulerDriver or create one more that extends MesosSchedulerDriver in test.